Update newbs_building_firmware.md : Guide users whose keymaps are in `.json` format. (#24663)
qmk/qmk_firmware: docs/newbs_building_firmware.md

Ψ Created a new keymap called <github_username> in: /home/me/qmk_firmware/keyboards/clueboard/66/rev3/keymaps/<github_username>.
```

> This is the location of your new `keymap.c` file. If your particular keyboards default keymap is a `.json` file there will be a `keymap.json` located in the generated folder instead of a `keymap.c` file. The `keymap.json` file can be converted to a `keymap.c` file using QMK's [`json2c`](cli_commands#qmk-json2c) utility. 

Reviewer: ```suggestion
This is the location of your new `keymap.c` file.

::: tip
If your keyboard default keymap is in JSON format, there will be a `keymap.json` in that location. The `keymap.json` file can be converted into `keymap.c` format using QMK's [`json2c`](cli_commands#qmk-json2c) utility.
::: tip
```

Consider using a tip box for this tip.

Reviewer 2: Honestly, "This is the location your new keymap file." may be better. It's more file type agnostic, and there  could simply be a reference that it may be a keymap.c or keymap.json, depending on the keyboard's default keymap.

Author: Thanks all for the suggestions and attention.

IMO as a new user, I saw that the next section, "Open `keymap.c` ....", referred to a file I did not have and this stopped me from moving forward. It seems more essential than a tip!

How about:
```
This is the location of your new keymap file. Your keyboards default keymap file may be a `.json` file or a `.c` file.  If your keymap is a `.json` file it can be converted to a `.c` file using QMK's [`json2c`](cli_commands#qmk-json2c) utility. 
```
